test/organizationMembership.matchers.spec.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const { expect } = require('./matchers');
+require('./organizationMemberships.matchers');
+
+describe('organizationMemberships.matchers', function(){
+  const validOrganizationMembership = {
+    uid: '56ff8cc9f22bbdf4f1721f149b30115c',
+    createdAt: '2020-12-09T20:04:13.320Z',
+    organizationApikey: 'WagonCorp',
+    memberUserDid: 'did:jlinc:XdsNStmHbQzSfaUx8f7r8BG-oJAnaXhoQeDApBJOrXs',
+    updatedAt: '2020-12-09T20:04:13.320Z',
+    admin: true,
+  };
+
+  it('anOrganizationMembership', function() {
+    expect({}).to.not.be.anOrganizationMembership();
+    expect(validOrganizationMembership).to.be.anOrganizationMembership();
+    expect({
+      ...validOrganizationMembership,
+      admin: undefined,
+      curator: true,
+    }).to.be.anOrganizationMembership();
+  });
+
+  it('anOrganizationCuratorMembership', function() {
+    expect(validOrganizationMembership).to.not.be.anOrganizationCuratorMembership();
+    expect({
+      ...validOrganizationMembership,
+      admin: undefined,
+      curator: true,
+    }).to.be.anOrganizationCuratorMembership();
```

Comment:
shouldn't each of these tests have something like
```js
expect(() => {
  expect(bad).to.be.aGood()
}).to.throw(`expected bad to be good`);
```

Reply:
aaah yeah good point, added to all tests